model/clusters_mgmt/v1/cluster_type.model

```diff
@@ -186,4 +186,7 @@ class Cluster {
 
 	// Node drain grace period.
 	NodeDrainGracePeriod Value
+
+	// Indicates whether that etcd is encrypted or not
+	EtcdEncryption Boolean
```

Comment:
Is there any chance that we will want or need to add other etcd related attributes in the future? If so I`d suggest to use a struct here:

```
struct Etcd {
  EncryptionEnabled Boolean
}
```

Or even two, one for general etcd things and another one for etcd encryption in particular:

```
struct Etcd {
  Encryption EtcdEncryption
}

struct EtcdEncription {
  Enabled Boolean
}
```

Reply:
I do not think so. @igoihman ?

Reply:
the current plan is to have it enabled or disabled. I'm not aware of other attributes we should support